tools/json_tools/gun_variant_validator.py

```diff
@@ -142,6 +142,7 @@
     "m1911a1_38super",
     "plasma_gun",
     "bbgun",
+    "460_xvr",
```

Comment:
Please please please don't ever add anything to these blacklists, fix the definitions so they pass.

Reply:
Sorry, I remember talking about how revolvers and such didn't really need the generic names and so I added it to the blacklist without a second thought